osx/make_app.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+# Usage: ./setup.py bdist_app
+# $1: wheel name
+# $2: package name
+set -e
```

Comment:
Maybe at `set -o nounset`, so the script bails out if you don’t use it properly, or it hits another error? e.g. `greadlink` isn’t present on my machine, so `$python3_bin_dir` is empty but the script continues regardless.